front/pages/w/[wId]/builder/assistants/index.tsx

```diff
@@ -105,14 +110,20 @@ export default function WorkspaceAssistants({
     agentsGetView: tabScope === "private" ? "list" : tabScope,
     includes,
   });
-  const [showDisabledFreeWorkspacePopup, setShowDisabledFreeWorkspacePopup] =
-    useState<string | null>(null);
 
-  const [assistantSearch, setAssistantSearch] = useState<string>("");
+  // except when user searches: search across all agents
```

Comment:
I don't understand the comment. My understanding of the code is more "if not searching, fetch nothing, otherwise, fetch them all"

Reply:
yes that's it, will improve com